_includes/footer-phone.html

```diff
@@ -0,0 +1,12 @@
+{% if page.layout == 'content' or page.layout == 'content-statements'  %}
+{% assign small = 'small' %}
+{% endif %}
+
+<div class="footer-phone {{ small }} ">
+<h2 class="footer-phone_text">{{ site.data.translations.footer-phone[page.lang] }}</h2>
+<h3 class="footer-phone_action_text">{{ site.data.translations.footer-phone-action[page.lang] }}</h3>
+
+<h4 class="footer-opening-hours">{{ site.data.translations.footer-phone-opening[page.lang] }}</h4>
```

Comment:
Zelfde als hierboven, dit is geen heading. 

Als je suggestie van hierboven overneemt, zou je hier `<h3>{{ site.data.translations.footer-phone-opening[page.lang] }}</h3>` kunnen doen (belangrijk dat het dan `h3` is, als die erboven `h2` wordt).